packages/resilience/internal.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
was this file added intentionally?

Reply:
yes this exposes the things under the /internal path.